src/main/kotlin/me/zeroeightsix/kami/BaritoneIntegration.kt

```diff
@@ -23,6 +22,9 @@ object BaritoneIntegration {
         }
     }
 
+    val mainProcess
+        get() = BaritoneAPI.getProvider().primaryBaritone.pathingControlManager.mostRecentInControl().kotlin
+
```

Comment:
I'd argue a better variable name is needed, such as `recentControlProcess`, as `mainProcess` could be mistaken for `primaryBaritone`